src/vs/workbench/contrib/chat/browser/media/chatViewWelcome.css

```diff
@@ -30,6 +30,8 @@
 		align-items: center;
 		justify-content: center;
 		overflow: hidden;
+		container-type: inline-size;
+		container-name: chat-welcome;
```

Comment:
ik container-name and `@container` is relatively new in css, so just for personal prefernce, let's maybe keep it simpler for now and not use it. 

`@media` is probably enough for what you want to do here, or maybe just giving it enough padding left/right might also be enough